--- Client.java	2013-12-19 14:53:23.000000000 -0500
+++ Client.java	2013-12-19 14:53:17.000000000 -0500
@@ -453,7 +453,18 @@
 		
 		return r.getResponseData().getResults();
 	}
-	
+
+    public List<Result> searchImagesByOffset(String query, int start)
+    {
+        Map<String, String> params = new LinkedHashMap<String, String>();
+
+        params.put("q", query);
+        params.put("start", Integer.toString(start));
+
+        Response r = sendImageSearchRequest(params);
+
+        return r.getResponseData().getResults();
+    }
 
 	public List<Result> searchLocal(double lat, double lon, String query)
 	{
